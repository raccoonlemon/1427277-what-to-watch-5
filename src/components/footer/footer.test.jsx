import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Footer from "./footer";

const renderer = new ShallowRenderer();

it(`<Footer> renders correctly`, () => {
  renderer.render(<Footer/>);
  const tree = renderer.getRenderOutput();
  expect(tree).toMatchSnapshot();
});
