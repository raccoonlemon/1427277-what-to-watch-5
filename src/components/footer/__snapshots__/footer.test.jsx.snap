// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> renders correctly 1`] = `
<footer
  className="page-footer "
>
  <Logo
    className="logo__link--light"
  />
  <div
    className="copyright"
  >
    <p>
      Â© 2019 What to watch Ltd.
    </p>
  </div>
</footer>
`;
